Clean up search path for test files to reflect /usr/local/share/ipptool instead
of the upstream /usr/share/cups/ipptool.

--- a/tools/ipptool.c
+++ b/tools/ipptool.c
@@ -496,10 +496,10 @@
 #endif /* _WIN32 */
                     )
 		{
-		  snprintf(filename, sizeof(filename), "%s/ipptool/%s", cg->cups_datadir, argv[i]);
+		  snprintf(filename, sizeof(filename), "%s/ipptool/%s", "/usr/local/share", argv[i]);
 		  if (access(filename, 0))
 		  {
-		    snprintf(filename, sizeof(filename), "%s/ipptool/%s.gz", cg->cups_datadir, argv[i]);
+		    snprintf(filename, sizeof(filename), "%s/ipptool/%s.gz", "/usr/local/share", argv[i]);
 		    if (access(filename, 0))
 		      strlcpy(filename, argv[i], sizeof(filename));
 		  }
@@ -683,7 +683,7 @@
 #endif /* _WIN32 */
           )
       {
-        snprintf(testname, sizeof(testname), "%s/ipptool/%s", cg->cups_datadir, argv[i]);
+        snprintf(testname, sizeof(testname), "%s/ipptool/%s", "/usr/local/share", argv[i]);
         if (access(testname, 0))
           testfile = argv[i];
         else
@@ -2463,10 +2463,10 @@
   if (*src == '<' && src[strlen(src) - 1] == '>')
   {
    /*
-    * Map <filename> to CUPS_DATADIR/ipptool/filename...
+    * Map <filename> to /usr/local/share/ipptool/filename...
     */
 
-    snprintf(dst, dstsize, "%s/ipptool/%s", cg->cups_datadir, src + 1);
+    snprintf(dst, dstsize, "%s/ipptool/%s", "/usr/local/share", src + 1);
     dstptr = dst + strlen(dst) - 1;
     if (*dstptr == '>')
       *dstptr = '\0';
